.github/workflows/run_tests.yml

```diff
@@ -66,17 +66,73 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: s-weigand/setup-conda@v1
+        with:
+          python-version: 3.7
       - uses: actions/setup-java@v1
         with:
           java-version: 1.8
       - uses: actions/setup-node@v1
-      - name: Run JS linter
+      - name: Install ord_schema
         run: |
           cd "${GITHUB_WORKSPACE}"
+          pip install -r requirements.txt
+          conda install -c rdkit rdkit
+          python setup.py install
+      - name: Install editor dependencies
+        run: |
+          cd "${GITHUB_WORKSPACE}/editor"
+          pip install flask
+          npm install google-protobuf puppeteer
+          wget https://storage.googleapis.com/ord-editor-test/editor_test_protobuf_1dae8fdd.tar
+          tar -xzf editor_test_protobuf_1dae8fdd.tar
+          wget https://github.com/google/closure-library/archive/v20200517.tar.gz
+          tar -xzf v20200517.tar.gz
+          git clone https://github.com/Open-Reaction-Database/ketcher.git
+      - name: Build editor JS
+        run: |
+          cd "${GITHUB_WORKSPACE}/editor"
+          make
+      - name: Run JS linter
+        run: |
+          cd "${GITHUB_WORKSPACE}/editor"
           # See https://github.com/google/closure-compiler/wiki/Lint-checks.
           npm install google-closure-compiler
+
+          # Run the closure compiler, having it lint code and produce warnings during compilation.
+          # In order to compile properly, we must provide all the source files necessary, 
+          # and make sure the compiler skips over files that aren't needed as dependencies.
+          # Because we are only checking code, we skip generating output and running optimizations.
+          # We also produce warnings only for our written source code 
+          # (by suppressing warnings for third-party libraries or machine generated code).
+
+          # Check reaction.js.
+          java -jar node_modules/google-closure-compiler-java/compiler.jar \
```

Comment:
Consider making this into a bash script that accepts an `entry_point` argument to avoid duplicating the code. See https://github.com/Open-Reaction-Database/ord-schema/blob/main/actions/commit_changes.sh that is used in `cleanup.yml`.

Reply:
Done; could you take another look?

Reply:
Very nice; thank you!